packages/legacy/core/App/components/modals/CommonRemoveModal.tsx

```diff
@@ -133,6 +133,10 @@ const CommonRemoveModal: React.FC<CommonRemoveModalProps> = ({ usage, visible, o
       return t('ContactDetails.RemoveContact')
     }
 
+    if (usage === ModalUsage.ContactRemoveWithCredentials) {
```

Comment:
Wondering if we should change to a `switch` statement since its all the same var?